app/src/main/java/com/github/se/project/ui/components/LessonEditor.kt

```diff
@@ -321,36 +322,58 @@ fun LessonEditor(
     Dialog(
         onDismissRequest = { showMapDialog = false },
         properties = DialogProperties(usePlatformDefaultWidth = false)) {
-          Column {
-            TimePicker(
-                state = timePickerState,
-            )
-            Button(onClick = { showTimeDialog = false }) { Text(stringResource(R.string.close)) }
-            Button(
-                onClick = {
-                  val selectedCalendar =
-                      Calendar.getInstance().apply {
-                        timeInMillis = currentDateTime.timeInMillis
-                        set(Calendar.HOUR_OF_DAY, timePickerState.hour)
-                        set(Calendar.MINUTE, timePickerState.minute)
-                      }
+          Surface(
+              shape = MaterialTheme.shapes.medium,
+              color = MaterialTheme.colorScheme.surfaceVariant,
+              modifier = Modifier.padding(16.dp)) {
+                Column(
+                    modifier = Modifier.padding(8.dp),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+                  TimePicker(state = timePickerState)
+
+                  Row(verticalAlignment = Alignment.CenterVertically) {
+                    Button(onClick = { showTimeDialog = false }) {
+                      Text(stringResource(R.string.close))
+                    }
```

Comment:
I personally quite liked the transparent design of the time picker, but I understand I'm in the minority. This version works well as well